src/sentry/constants.py

```diff
@@ -222,6 +222,7 @@ def get_all_languages():
     "sentry.rules.conditions.tagged_event.TaggedEventCondition",
     "sentry.rules.conditions.event_frequency.EventFrequencyCondition",
     "sentry.rules.conditions.event_frequency.EventUniqueUserFrequencyCondition",
+    "sentry.rules.conditions.event_frequency.SessionPercentCondition",
```

Comment:
We should probably only display this behind a feature flag, at least initially.

Also wondering if we should only display this for orgs that have sessions

Reply:
Yeah you're right. I'll figure out how we want to feature-gate this and add that in

Reply:
I vaguely remember inter jeff did this by adding in an if statement into the endpoints that return the list of Conditions. I feel like @davidenwang might have also done something similar

Reply:
yup I think the easiest way to do this is add an if statement into the `ProjectRulesConfigurationEndpoint` to check for the feature flag and skip over adding the condition to the Response.

Reply:
Thanks for the tip. Implemented it that way and created the flag `organizations:issue-percent-filters`